app/setup.py

```diff
@@ -264,7 +264,7 @@ def setup_secure_headers(application):
         # browsersync is configured to bind on port 5075
         csp_policy["connect-src"] += ["ws://localhost:35729"]
 
-    application.config["SESSION_COOKIE_SAMESITE"] = "Strict"
+    application.config["SESSION_COOKIE_SAMESITE"] = "Lax"
```

Comment:
I think it's one of the requirements on the card: "Ensure samesite cookie policy change (to Lax) is in place"

Reply:
FYI, `Strict` does technically work now that we have custom domains. Lax cookies are sent with cross-site requests, and Strict cookies are not. So `Strict` when launching from the same parent domain works and works as intended, but means we can't launch from non-matching domains like the default Cloud Run URL or localhost since parent domains are different. So it is more to work out how `Strict` worked when on k8s with localhost etc when it shouldn't have?.

So technically for envs maybe apart from `Staging` should be `Strict`. I think card stated that, so it can be investigated separately. We could set it to Strict now and accept env based functional tests won't work for now until we investigate or make it configurable. I think it is appropriate to progress this as `Strict` and investigate separately about launching from localhost etc. 

Thoughts?

Reply:
I wasn't at that tech session. No worries, we can discuss it at the next one.